cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -641,6 +643,9 @@ var acceptableErrors = []error{
 	order.ErrCannotValidateAsset,         // Is thrown when attempting to get order limits from an asset that is not yet loaded
 	order.ErrCannotValidateBaseCurrency,  // Is thrown when attempting to get order limits from an base currency that is not yet loaded
 	order.ErrCannotValidateQuoteCurrency, // Is thrown when attempting to get order limits from an quote currency that is not yet loaded
+	account.ErrExchangeHoldingsNotFound,  // Is thrown when an exchange's holdings are not found
+	ticker.ErrNoTickerFound,              // Is thrown when no ticker is found
+	orderbook.ErrCannotFindOrderbook,     // Is thrown when no orderbook is found
```

Comment:
Nit, but you've run into it:
Seem to have created a copy/paste paradigm here for "Is thrown when X", when it seems like that's implicit for all expected errors.
Also don't think we're adding anything with the comments for these 3. Aren't they saying the same thing as the error constant.